jumpscale/packages/codeserver/package.py

```diff
@@ -1,9 +1,11 @@
 from jumpscale.god import j
+import os
 
 
 class codeserver:
     def __init__(self):
         self.bin_path = j.sals.fs.join_paths(j.core.dirs.BINDIR, "code-server")
+        self.script_path = j.sals.fs.join_paths(os.path.dirname(__file__), "codeserver-install.sh")
```

Comment:
I think calling it in this `__init__` function is not OK ? It may cause the package to corrupt and cause deletion error also.
 maybe in install method or `__init__.py` if you wanna call it in docker file and here too, will be something like  `j.packages.codeserver.install()`

Reply:
I don't call anything here. I just get the path for the script!

Reply:
okay then fine (y)